controllers/api/openstack/rename.py

```diff
@@ -0,0 +1,30 @@
+import cherrypy
```

Comment:
Please group and order imports as per [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports).

Reply:
I presume you are referring specifically to the following?

> Imports should be grouped in the following order:
> 1. standard library imports
> 2. related third party imports
> 3. local application/library specific imports
>
> You should put a blank line between each group of imports.